src/lib.rs

```diff
@@ -31,7 +31,9 @@ fn simulate_data_reception(x: i32, y: i32) -> Frame {
 
     Frame {
         sprite: Some(sprite),
-        sound: Some(sound),
+        sound: None,
+        // At the moment we are only sending sprite data, if you want to send sound data, uncomment the next line and comment the previous one
+        //sound: Some(sound),
```

Comment:
Why are only sprites being sent?

![image](https://github.com/Pending-Name-21/console/assets/64939916/0058d468-ee0a-4297-bc2c-e3d9d4065bfa)

Reply:
Yes, it is possible to send only one Frame composed only of Sprite or only Sound or both. In this example only Sprite is being sent, in any case leave a comment in the code.

Reply:
This is because of the data simulation right?

Reply:
Yes